### ISSUE TITLE
No Auth Request for Google Drive - regular ipythonic notebook server?

### ISSUE THREAD

**Issue by kastman** [jupyter/jupyter-drive#107]
Hi all,
I'm sure I'm missing something basic, but I was able to successfully clone, install a config and run ipython with the jupyterdrive profile, but when I open it I get a standard ipythonic server, and no auth request for Google Drive. Is there an additional config step not in the readme that I need to do? Sorry this is so basic, but I've been looking for an hour and can't find it. Using py3 and ipython 3
python -c 'import IPython; print(IPython.sys_info())'

Thanks for any suggestions,
Erik

```
{'commit_hash': 'cbccb68',
 'commit_source': 'installation',
 'default_encoding': 'UTF-8',
 'ipython_path': '/Users/kastman/miniconda/envs/py3/lib/python3.4/site-packages/IPython',
 'ipython_version': '3.1.0',
 'os_name': 'posix',
 'platform': 'Darwin-14.3.0-x86_64-i386-64bit',
 'sys_executable': '/Users/kastman/miniconda/envs/py3/bin/python',
 'sys_platform': 'darwin',
 'sys_version': '3.4.3 |Continuum Analytics, Inc.| (default, Mar  6 2015, '
                '12:07:41) \n'
                '[GCC 4.2.1 (Apple Inc. build 5577)]'}
```

**Comment by Carreau** [comment]
Hum, I'll clean my system and try the install instruction, things have changed a bit the last few weeks. 

Do you get an error in the notebook terminal, or in the browser JS console ?

**Comment by kastman** [comment]
Nope, no errors at all, either in terminal or through the browser. It seems to work exactly like a standard filesystem-based notebook server.

I did give permissions to coLaboratory (the older version?) - is it possible that there's a check+setup that's being skipped? I don't think this is the case because I was sitting side-by-side with another person who also had this issue and hadn't given permissions, but I thought I'd mention it.

**Comment by Carreau** [comment]
No coLaboratory should not affect that at all. can you `$ cat ~/.ipython/profile_default/ipython_notebook_config.json` ? and maybe hop on gitter to debug this live ?

**Comment by kastman** [comment]
I think this may be a doc issue more than anything else - when I went back to try it again and forgot to use a specific profile (i.e. `ipython notebook` instead of `ipython notebook --profile jupyter-drive`) I got the authentication request and I believe it's working now. From the readme I thought you had to specifically request the drive add-on with a profile, but I guess that's not the case? 

Thanks for your help, I'm closing this issue now. Just an FYI that I was confused, so maybe clarifying the steps for an initial start would help?
